src/fragments/Graph.react.js

```diff
@@ -1,9 +1,42 @@
 import React, {Component} from 'react';
-import {clone, equals, filter, has, includes, isNil, omit, type} from 'ramda';
+import ResizeDetector from 'react-resize-detector';
+import {
+    clone,
+    equals,
+    filter,
+    has,
+    includes,
+    isNil,
+    mergeDeepRight,
+    omit,
+    type,
+} from 'ramda';
 import PropTypes from 'prop-types';
 import {graphPropTypes, graphDefaultProps} from '../components/Graph.react';
 /* global Plotly:true */
 
+/**
+ * `autosize: true` causes Plotly.js to conform to the parent element size.
+ * This is necessary for `dcc.Graph` call to `Plotly.Plots.resize(target)` to do something.
+ *
+ * Users can override this value for specific use-cases by explicitly passing `autoresize: true`
+ * if `responsive` is not set to True.
+ */
+const DEFAULT_LAYOUT = {
+    autosize: true,
+};
+
+/**
+ * `responsive: true` causes Plotly.js to resize the graph on `window.resize`.
+ * This is necessary for `dcc.Graph` call to `Plotly.Plots.resize(target)` to do something.
+ *
+ * Users can override this value for specific use-cases by explicitly passing `responsive: false`
+ * if `responsive` is not set to True.
+ */
+const DEFAULT_CONFIG = {
+    responsive: true,
+};
+
```

Comment:
When responsive is true, whether through true or auto, will apply these values to config/figure to get the desired behavior.